lualib/rspamadm/decrypt_header.lua

```diff
@@ -0,0 +1,46 @@
+local util = require 'lua_util'
+local argparse = require 'argparse'
+
+local parser = argparse()
+    :name "rspamadm decrypt header"
+    :description "Decrypt given header with given key and nonce"
+    :help_description_margin(32)
+
+parser:option "-H --header"
+      :description("Encrypted header")
+      :argname("<header>")
+parser:option "-k --key"
+      :description("Key used to encrypt header")
+      :argname("<key>")
+parser:option "-n --nonce"
```

Comment:
It looks good, but can we just use a concatenation of `nonce` + `encrypted data` as we will likely have? Of course we should have an ability to decrypt/encrypt with/without embedded nonce.